src/config/ssd.md

```diff
@@ -0,0 +1,149 @@
+# Solid State Drives
+
+Post installation, you will need to enable TRIM for solid state drives. You can
+check which devices allow TRIM by running:
+
+```
+$ lsblk --discard
+```
+
+If the DISC-GRAN (discard granularity) and DISC-MAX (discard maximum bytes)
+columns are non-zero, that means the block device has TRIM support. If your
+solid state drive partition does not show TRIM support, please verify that you
+chose a file system with TRIM support (ext4, btrfs, etc.).
+
+To run TRIM one-shot, you can run
+[`fstrim(8)`](https://man.voidlinux.org/fstrim.8) manually. For example, if your
+/ directory is on an SSD:
+
+> Note: To run fstrim on F2FS file systems, you need to be running kernel 4.19
+> or greater.
```

Comment:
We've recently updated [the style guide with a policy on notes](https://docs.voidlinux.org/contributing/void-docs/style-guide.html#notes), so this text should not use blockquoting.

To not interrupt the flow, i'd like this sentence to immediately follow "manually.", and begin with: "Note that to run \`fstrim\` on ..."